.devcontainer/home_assistant_run.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/with-contenv bashio
+# ==============================================================================
+# Start Home Assistant service
+# This file was taken from:
+# https://github.com/home-assistant/core/blob/dd6725b80a5efebda36c64c78250f3374e85c3a7/rootfs/etc/services.d/home-assistant/run
+# And modified to allow starting the Python Debugger.
+# ==============================================================================
+
+# shellcheck shell=bash
+
+cd /config || bashio::exit.nok "Can't find config folder!"
+
+# Enable mimalloc for Home Assistant Core, unless disabled
```

Comment:
Why?

Reply:
This file was taken from here: https://github.com/home-assistant/core/blob/2e81be77216315c31c38334d912beca55a7197ad/rootfs/etc/services.d/home-assistant/run

I just made my modifications, and plan to send a PR to upstream in a later moment.